wrangler/package.json

```diff
@@ -9,6 +9,7 @@
     "preinit": "[ \"${CI}\" = true ] && npx envdist -e || npx envdist"
   },
   "dependencies": {
+    "@cloudflare/wrangler": "^1.19.0",
```

Comment:
We've gone this route in the past and having wrangler install globally has worked better so we won't commit this